pkg/tests/end_to_end_tests/fixtures/jaeger_binaray_tag/trace.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I think we should import all tests, let me do it as part of https://github.com/timescale/promscale/pull/1627

Reply:
Once we merge https://github.com/timescale/promscale/pull/1627/files in which Arun imports all the Jaeger integration tests we can clean this fixtures and the test related code added by this PR

Reply:
I deleted the e2e test and added some unittests instead